ideascube/mediacenter/tests/test_views.py

```diff
@@ -10,7 +10,32 @@
 from .factories import DocumentFactory
 from ideascube.search.models import Search
 
-pytestmark = pytest.mark.django_db
+# Almost every call to the mediacenter view need a catalog.
```

Comment:
The "almost" is not true anymore now we do not display package_id filter in the views.
So, indeed, no need to mock the catalog all the time.

I will change this.